README.md

```diff
@@ -1,14 +1,31 @@
 # Welcome to the Windows Bridge for iOS Project
 The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices by re-using your Objective-C code and iOS APIs alongside Windows 10 features like Cortana and Windows Notifications.
 
+|                      |           Stable (master)                        |            Pre-release (develop)          |
+| -------------------: | :----------------------------------------------: | :---------------------------------------: |
+|           **Build**  |                  -                               |       ![VSTS][develop-build-badge]        |
+|  **GitHub Release**  |[![Github Release][github-rel-badge]][github-rel] |               -                           |
+|    **WinObjC.Tools** |     [![Tools][choco-badge]][choco]               |   [![Tools][choco-badge-pre]][choco]      |
+| **WinObjC.Language** |[![Language][nuget-lang-badge]][nuget-lang]       |[![Language][nuget-lang-badge-pre]][nuget-lang]   |
+|**WinObjC.Frameworks**|[![Frameworks][nuget-frmwrk-badge]][nuget-frmwrk] |[![Frameworks][nuget-frmwrk-badge-pre]][nuget-frmwrk]|
+
 ## Download and Installation
 To use the bridge you'll need:
-- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest). Click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice (example: C:\winobjc) when complete.
 - **Windows 10**, build 10586 or higher. Validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system).
-- **Visual Studio 2015 Update 3** with Windows developer tools. Visual Studio 2015 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
+- **Visual Studio 2017** with Windows developer tools. Visual Studio 2017 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
    1. Programming Languages -> Visual C++
    2. Universal Windows App Development Tools (all)
```

Comment:
Okay, this is the Download and Installation section. Seems like this should be more of a "Add WinObjC.Framworks package reference to your project" section.
The full list of VS install options for WinObjC development should be included under the fancy Advanced Installation for Contributors and Ninjas section.

---
In reply to: [105712228](https://github.com/Microsoft/WinObjC/pull/2211#discussion_r105712228) [](ancestors = 105712228)